Sources/SwiftDocCUtilities/ArgumentParsing/ActionExtensions/ConvertAction+CommandInitialization.swift

```diff
@@ -72,7 +71,7 @@ extension ConvertAction {
             htmlTemplateDirectory: convert.templateOption.templateURL ?? fallbackTemplateURL,
             emitDigest: convert.emitDigest,
             currentPlatforms: parsedPlatforms,
-            buildIndex: convert.index,
+            buildIndex: convert.emitLMDBIndex || convert.index,
```

Comment:
Shouldn't we always build an index now? Not sure that the `buildIndex` field of the ConvertAction is used anymore so it might make sense to remove it (and modify the initialisers accordingly).

Reply:
Unfortunately this is public API now so we need to leave the old `buildIndex` property for at least some time for compatibility but I agree this is confusing.

We could definitely refactor this now (deprecate this initializer and create a new one) but I wasn't sure if it was worth the churn for this PR.

Reply:
OK, maybe the `buildIndex` argument then deserves a doc comment saying that it actually means build the LMDB index?

Reply:
Good idea. Updated here: 61afd3d.

Reply:
Good idea. Updated here: 61afd3d.